framework_tvb/tvb/tests/framework/core/neotraits/db_test.py

```diff
@@ -64,7 +64,7 @@ class FooIndex(HasTraitsIndex):
 
     array_int_max = Column(Integer)
     array_int_min = Column(Integer)
-    array_int_mean = Column(Integer)
+    array_int_mean = Column(Float)
```

Comment:
It makes sense. Shouldn't we also have the `ensure_float` where this field is computed ?

Reply:
I think it is not necessary, because according to numpy.mean official documentation ( https://docs.scipy.org/doc/numpy/reference/generated/numpy.mean.html ), the result type for integer inputs is float64.
Yet, if it is preferred, we can add this ensure_float in the place where this field is computed, for extra safety.

Reply:
Good. Then, it is not necessary to put `ensure_float`. Leave it as you initially proposed.